### PR TITLE
Added types for rrdtool

### PR DESCRIPTION
**Pull request by filiptypjeu**
Please fill in this template.

- [ x ] Use a meaningful title for the pull request. Include the name of the package modified.
- [ x ] Test the change in your own code. (Compile and run.)
- [ x ] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [ x ] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [ x ] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [ x ] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If adding a new definition:
- [ x ] The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`
- [ x ] If this is for an npm package, match the name. If not, do not conflict with the name of an npm package.
- [ x ] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [ x ] Represents shape of module/library [correctly](https://www.typescriptlang.org/docs/handbook/declaration-files/library-structures.html)
- [ x ] `tslint.json` [should contain](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#linter-tslintjson) `{ "extends": "@definitelytyped/dtslint/dt.json" }`, and no additional rules.
- [ x ] `tsconfig.json` [should have](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#tsconfigjson) `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

### CODE REVIEW

**Comment by DangerBotOSS** [comment]
Inspecting the JavaScript source for this package found some properties that are not in the .d.ts files.
The check for missing properties isn't always right, so take this list as advice, not a requirement.
## rrdtool ([<kbd>unpkg</kbd>](https://unpkg.com/browse/rrdtool@latest/))
was missing the following properties:
1. open
1. create
1. now
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against e168251e13a64b7d9e91ffa05701cbcd83642a23
</p>

File: types/rrdtool/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+     */
+    function open(file: string): RrdtoolDatabase;
+
+    /**
+     * Returns the current unix timestamp.
+     */
+    function now(): number;
+}
+
+export default rrdtool;
```
Comment (MysteryBlokHed) [inline types/rrdtool/index.d.ts:142]: You could probably move these functions out of a namespace and `export` them separately instead:
```suggestion
/**
 * Creates a new database.
 */
export function create<D extends RrdtoolData>(
    file: string,
    opts: {
        step?: Duration;
        start?: Timestamp;
        force?: boolean;
    },
    args: ReadonlyArray<RrdtoolArgument<D>>,
): RrdtoolDatabase<D>;

/**
 * Loads an existing database.
 */
export function open(file: string): RrdtoolDatabase;

/**
 * Returns the current unix timestamp.
 */
export function now(): number;
```

Comment (filiptypjeu) [reply types/rrdtool/index.d.ts:142]: Sure! How do I add a default export in that case?

Comment (sandersn) [reply types/rrdtool/index.d.ts:142]: I don't think you should have a default export in this case. Looking at the code as shipped at https://unpkg.com/rrdtool, there's no `export.default = ...` property, so people who don't have "esModuleInterop": true will not be able to use a default import.

Consumers of the package who want to use `import rrdtool from 'rrdtool'` will need to add `"esModuleInterop": true`, but Definitely Typed tests need to be backward compatible to show that the types will work with old code bases.

File: types/rrdtool/rrdtool-tests.ts [hunk 2]
```diff
@@ -0,0 +1,138 @@
+import rrdtool, { DataSource, RoundRobinArchive, RrdtoolDatabase } from 'rrdtool';
```
Comment (sandersn) [inline types/rrdtool/rrdtool-tests.ts:1]: there's no default export in the shipped code, so you shouldn't assert that one will be available.

File: types/rrdtool/tslint.json [hunk 3]
```diff
@@ -0,0 +1,14 @@
+{
+    "extends": "@definitelytyped/dtslint/dt.json",
+    "rules": {
+        "npm-naming": [
+            true,
+            {
+                "mode":"code",
+                "errors": [
+                    ["NoDefaultExport", false]
```
Comment (sandersn) [inline types/rrdtool/tslint.json:9]: without the `export default`, you shouldn't need to disable this rule either.

**Review by sandersn — CHANGES_REQUESTED** [review]

File: types/rrdtool/index.d.ts (hunk 1 above)
Comment (filiptypjeu) [reply types/rrdtool/index.d.ts]: Aah okay, thanks for the explanation!

File: types/rrdtool/tslint.json (hunk 3 above)
Comment (peterblazejewicz) [reply types/rrdtool/tslint.json]: amended

File: types/rrdtool/rrdtool-tests.ts (hunk 2 above)
Comment (peterblazejewicz) [reply types/rrdtool/rrdtool-tests.ts]: amended

**Review by peterblazejewicz — APPROVED** [review]
LGTM!
@filiptypjeu thanks!

**Review by sandersn — APPROVED** [review]

**Review by MysteryBlokHed — APPROVED** [review]

**Comment by filiptypjeu** [comment]
Ready to merge

**Pull request merged**